website/views/shared/modals/members.jade

```diff
@@ -62,14 +62,14 @@ script(type='text/ng-template', id='modals/private-message.html')
 
 script(type='text/ng-template', id='modals/send-gift.html')
   .modal-header
-    h4 Send Gift to {{::profile.profile.name}}
+    h4=env.t('sendGiftHeading', {name: "{{profile.profile.name}}"})
```

Comment:
I'm sorry about this. This issue was a bit tricky for me, so I appreciate your help @crookedneighbor 
May I change this back and push the changes now?